code/__DEFINES/~yogs_defines/reactions.dm

```diff
@@ -1,5 +1,5 @@
 //Cold Fusion
-#define FUSION_TEMPERATURE_THRESHOLD_MINIMUM 4000 // This is the minimum possible required temperature that dilithium can lower fusion to.
+#define FUSION_TEMPERATURE_THRESHOLD_MINIMUM 425 // This is the minimum possible required temperature that dilithium can lower fusion to.
 #define DILITHIUM_LAMBDA 0.0087 // Affects how much Dilithium is required to get the required fusion temperature down to FUSION_TEMPERATURE_THRESHOLD_MINIMUM
```

Comment:
Take these //comments and make them /// and put them one line above the defines

Reply:
done, also autodocced the BZ stuff